test/truffle/compiler/pe/core/binding_pe.rb

```diff
@@ -15,7 +15,7 @@
 example "x = 14; p = Proc.new { }; p.binding.local_variable_get(:x)", 14
 
 # set + get
-tagged example "b = binding; b.local_variable_set(:x, 14); b.local_variable_get(:x)", 14
+example "b = binding; b.local_variable_set(:x, 14); b.local_variable_get(:x)", 14
```

Comment:
Could you add a few extra PE examples in `eval_pe.rb` to show what this PR improves?